apps/dashboard/app/views/extended_pages/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= javascript_include_tag 'dashboard', nonce: true %>
```

Comment:
can we reuse the dashboard's partial entirely?

Reply:
Currently we can, but I guess it introduces a dependency to the `dashboard` that we do not want.

Looking at the `dashboard.js` file, the content is mainly for the XDMoD widget.
The last function, to add an error message for links that open in a new window, could apply to all widgets. But not sure if it was developed for XDMoD too.

We could refactor and add these functions to a new widgets.js file and and include this one in both `dashboard` and `customs pages`

Reply:
Yea I'd say we could put it into `views/shared`, but the javascript portion complicates it. We can defer it.